_data/kaupungit.yml

```diff
@@ -263,3 +353,23 @@
   logo: "seinajoki.png"
   irc: "#SjkHacklab"
   irc_network: "Freenode"
+
+- kaupunki: Kajaani Hacklab
```

Comment:
General consensus is that Kajaani doesn't mainly either exist or doesn't want to collaborate with us, so remobe whole Kajaani. In the chance they sometimes answer to us then we'll think again.

Reply:
done in 9973a83